<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialog</class>
 <widget class="QWidget" name="InputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-20</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(32, 32, 35, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_KeyBoard">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>428</width>
     <height>572</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/DigitalKeyBoard.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_Default">
   <property name="geometry">
    <rect>
     <x>706</x>
     <y>526</y>
     <width>217</width>
     <height>72</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_Value">
   <property name="geometry">
    <rect>
     <x>832</x>
     <y>248</y>
     <width>217</width>
     <height>72</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_Confirm">
   <property name="geometry">
    <rect>
     <x>1012</x>
     <y>526</y>
     <width>217</width>
     <height>72</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
